README.md

```diff
@@ -138,28 +151,27 @@ Congratulations! You have successfully started the flow. Carry on reading the ne
 - Handle callbacks
 - Remove the SDK from the page
 - Customise the SDK
-- Upload files to the Onfido API
 - Create checks
 
 ## Handling callbacks
 
-A number of callback functions are fired at various points of the flow. The most important function is `onComplete`. Inside this callback, you would typically send the captured images to your backend server, where you would then [upload files](#uploading-files-to-onfido-api) and [create checks](#creating-checks) using the [Onfido API](https://documentation.onfido.com/).
+A number of callback functions are fired at various points of the flow. The most important function is `onComplete`. Inside this callback, you would typically [create checks](#creating-checks) using the [Onfido API](https://documentation.onfido.com/) on your backend server.
```

Comment:
We should make it more clear you should not actually create a check from the client. e.g. "Inside this callback you would typically trigger your backend to create a check using the Onfido API."